### DISCUSSION TITLE
홈 화면 커버 사진을 제거하고 싶은데 어떻게 하는 걸까요?

### DISCUSSION THREAD

**Discussion by zoohee** [pronist/hELLO#220] [Q&A] [answered]
홈화면이 '리스트' 타입이고 블로그 들어가자마자 맨위에 '전체 글'이라고 뜨는 부분 배경사진을 단색으로 변경 또는 제거하고 싶습니다! HTML이나 CSS 코드 뒤져봐도 잘 모르겠더라구요.. 혹시 어느 부분을 건드려야할지 알려주실 수 있을까요?

**Comment by pronist** [comment, marked as answer]
@zoohee 안녕하세요. 답변이 많이 늦었습니다.

리스트에서 표기되기는 홈 화면 이미지를 제거하려면 `s_list_image` 를 찾아서 지워주시면 되겠습니다. 아래의 코드는 원본 소스에서 나타나는 모양이므로 `skin.html` 에서는 HTML Tag 로 보일 것입니다.

https://github.com/pronist/tistory-skin-hELLO/blob/9fa166f2cf8164c9302648c5770146027d3b669c/views/Main/List.pug#L4-L9

**Reply by zoohee** [reply]
감사합니다 ! :)
